### ISSUE TITLE
How to integrate jstorrent video player into html ?

### ISSUE THREAD

**Issue by albabosh** [kzahel/jstorrent-legacy-app#54]
Is it possible now to integrate jstorrent with a weeb player into html page ?

**Comment by kzahel** [comment]
no, because it needs lots of things that the chrome.\* APIs provide, filesystem, sockets, etc.

**Comment by kzahel** [comment]
However, it is possible to do this, if the user has the app installed. It's also possible to do an inline install (from the webpage)

**Comment by kzahel** [comment]
Great question — not supported yet, but it would be really cool to support in the future. I've rebuilt JSTorrent from scratch. Get it at **[jstorrent.com](https://jstorrent.com)**

If you're still interested in this, please reach out!
